libs/scully/src/lib/utils/handlers/handleRouteDiscoveryDone.ts

```diff
@@ -9,6 +9,16 @@ export async function handleRouteDiscoveryDone(handledRoutes: HandledRoute[]) {
   performance.mark('startRouteDonePlugins');
   performanceIds.add('RouteDonePlugins');
   const clone = deepClone(handledRoutes);
-  await Promise.all(Object.values(plugins.routeDiscoveryDone).map((plugin) => plugin(clone)));
+
+  const plugins_routeDiscoveryDone = plugins.routeDiscoveryDone;
+
+  const plugins_routeDiscoveryDone_values = Object.values(plugins_routeDiscoveryDone);
+
+  const arrPlugins = plugins_routeDiscoveryDone_values.map((plugin) => {
+    return plugin(clone);
+  });
+
+  await Promise.all(arrPlugins);
+
```

Comment:
For the same reason, this code is unneeded

Reply:
This was for my internal test, just so I could be sure what was happening here.  Didn't mean for that to be committed.